src/app/components/our-team/our-team.component.html

```diff
@@ -1,24 +1,24 @@
 <app-header-static></app-header-static>
 <div class="container-top">
 	<div class="our-team-container ">
-	<div class="our-team-title">
-		{{coreTeamType}}
-	</div>
+        <div class="our-team-title">
+		    {{coreTeamType}}
+        </div>
         <div class="row">
             <h3 ng-if="noTeamDisplay" class="team-heading">{{noTeamDisplay}}</h3>
             <!-- This card is for Core Team -->
             <div class="col-lg-3 col-md-3 col-sm-12" *ngFor="let data of coreTeamDetails">
                 <div class="team-card">
                     <div class="team-link-head">
-                        <span>
-                <a ng-if="data.linkedin_url !== 'null'" href="{{data.linkedin_url}}" target="_blank"><i class="fab fa-linkedin"></i></a>
-            </span>
-                        <span>
-                <a ng-if="data.github_url !== 'null'" href="{{data.github_url}}" target="_blank"><i class="fab fa-github"></i></a>
-            </span>
-                        <span class="right link-team-user">
-                <a ng-if="data.personal_website !== 'null'" href="{{data.personal_website}}" target="_blank">Personal Website</a>
-            </span>
+                        <span class="col-md-1 col-sm-1 col-xs-1">
+                            <a ng-if="data.linkedin_url !== 'null'" href="{{data.linkedin_url}}" target="_blank"><i class="fab fa-linkedin"></i></a>
```

Comment:
Isn't it still 80+ length. Also, HTML tags are not line by line.

Reply:
@Shekharrajak done, please take a look if it looks good to you